### ISSUE TITLE
state if an app is not present in singularity inspect --app

### ISSUE THREAD

**Issue by jscook2345** [apptainer/singularity#3051]
## Version of Singularity:

3.1.0

<pre>
[justin@ip-172-31-94-96 inspect]$ singularity inspect --app bob alpine_latest.sif
WARNING: labels metadata was not found.

[justin@ip-172-31-94-96 inspect]$ singularity inspect --app bob -e alpine_latest.sif
WARNING: environment metadata was not found.
</pre>

If you specify an app that does not exist, I would like to see an error saying that instead of trying to find the metadata.

**Comment by phpHavok** [comment]
This appears to be fixed in `3.2.0-rc1`.

```
[jacob@gideon temp]$ singularity inspect --app bob alpine_latest.sif
FATAL:   exit status 2: App bob does not exist.
[jacob@gideon temp]$ singularity inspect --app bob -e alpine_latest.sif
FATAL:   exit status 2: App bob does not exist.
```

**Comment by WestleyK** [comment]
Closing. Confirmed to be fixed in master.
